### DISCUSSION TITLE
Request to Add GXDE OS Image to Your VM Application

### DISCUSSION THREAD

**Discussion by SunnyPai0413** [xoureldeen/Vectras-VM-Android#150] [Ideas]
Dear Vectras Team,

I hope this message finds you well. I am writing on behalf of GXDE OS, a Debian-based distribution featuring the Deepin desktop environment and the aptss package manager. We are interested in having our OS image included in your VM application and would like to know the necessary steps and requirements for this process.

GXDE OS supports multiple architectures, including amd64, arm64, and LoongArch. We can provide you with links to our image repositories to facilitate the integration.

Could you please let us know what specific information or resources you need from us to proceed? We are eager to collaborate and ensure a smooth addition of our distribution to your platform.

Thank you for your time and assistance. We look forward to your response.

Best regards,

SunnyPai
DevOps Engineer
GXDE OS Team

**Comment by SunnyPai0413** [comment]
@xoureldeen

**Comment by xoureldeen** [comment]
hello @SunnyPai0413 just give us qcow2 file in downloadable url like github or mediafire without quota like g drive and provide the icon or logo of the os and required qemu params for this image,
currently we accept x86_64 and i386 and ppc and arm64, thanks.

**Comment by SunnyPai0413** [comment]
Dear Vectras Team,

I apologize for the delay in completing the image upload due to slow internet speeds at the hotel. I have now successfully uploaded the image and submitted it via the issue tracker. Therefore, I will be closing this discussion here.

Thank you for your understanding.

With appreciation,
SunnyPai
